doc/api/util.md

```diff
@@ -369,9 +369,23 @@ Returns the string name for a numeric error code that comes from a Node.js API.
 The mapping between error codes and error names is platform-dependent.
 See [Common System Errors][] for the names of common errors.
 
+## `util.getSystemErrorMap()`
+<!-- YAML
+added: v9.7.0
```

Comment:
```suggestion
added: REPLACEME
```

Reply:
@richardlau  What version should I put here? or leave it like that ?

Reply:
Put `REPLACEME` -- it's a placeholder value that will get replaced when a release is cut.